bundle/Helper/ImportUrlsHelper.php

```diff
@@ -0,0 +1,250 @@
+<?php
+
+namespace Novactive\Bundle\eZSEOBundle\Helper;
+
+use Doctrine\ORM\EntityManagerInterface;
+use eZ\Publish\Core\SignalSlot\URLWildcardService;
+use Novactive\Bundle\eZSEOBundle\Entity\RedirectImportHistory;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Component\Filesystem\Filesystem;
+
+
+/**
+ * Helper for import url.
+ */
+class ImportUrlsHelper
+{
+    /**
+     * @var URLWildcardService
+     */
+    private $urlWildCardService;
+
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    /**
+     * @var EntityManagerInterface
+     */
+    private $entityManager;
+
+    /**
+     * @var Filesystem
+     */
+    private $fs;
+
+    private $webDirectory;
+
+    /**
+     * ImportUrlsHelper constructor.
+     * @param URLWildcardService $urlWildcardService
+     * @param EntityManagerInterface $entityManager
+     * @param $webDirectory
+     * @param TranslatorInterface $translator
+     * @param LoggerInterface $logger
+     * @param Filesystem $fileSystem
+     */
+    public function __construct(
+        URLWildcardService $urlWildcardService,
+        EntityManagerInterface $entityManager,
+        $webDirectory,
+        TranslatorInterface $translator,
+        LoggerInterface $logger,
+        Filesystem $fileSystem
+    ) {
+        $this->urlWildCardService = $urlWildcardService;
+        $this->entityManager = $entityManager;
+        $this->webDirectory = $webDirectory;
+        $this->translator = $translator;
+        $this->logger = $logger;
+        $this->fs = $fileSystem;
+
+    }
+
+    /**
+     * @param $filePath
+     * @return array
+     */
+    public function importUrlRedirection($filePath)
```

Comment:
we should use the PHPSpreadsheet bundle to handle more than CSV

Reply:
for the moment, it is not necessary to use PHPSpreadsheet since the current need requires us to import only for .csv files